buttons/buttons.css

```diff
@@ -2249,3 +2249,73 @@ a:focus-visible {
 }
 /* Deepesh Sharma's button 1 ends*/
 
+/* kkartik07's 2nd btn start */
+.kkartik07-btn-2 {
+    height: 5em;
+	width: 12em;
+	border-top: 3px solid rgb(209, 203, 203);
+	border-right: 3px solid #CCC;
+	border-left: 4px solid #777;
+	border-bottom: 4px solid #777;
+	border-radius: 1em;
+	margin: 3em;
+	background-color: #999;
+	box-shadow:  -.1em .1em 0 #000,
+				 -.2em .2em 0 #000,
+				 -.3em .3em 0 #000,
+				 -.4em .4em 0 #333,
+				 -.5em .5em 0 #333,
+				 -.6em .6em 0 #333,
+				 -.7em .7em 0 #333,
+				 -.9em .9em 0 #333,
+				 -1em 1em 0 #333,
+				 -1.1em 1.1em 0 #333,
+				 -1.2em 1.2em 0 #333;
+	transition: box-shadow 0.4s,
+				border-top 0.2s,
+				border-right 0.2s;
+}
+
+.kkartik07-btn-2:hover, .kkartik07-btn-2:focus {
+    border-top: 3px solid rgba(119, 119, 119, 0.699);
+    border-right: 3px solid rgba(119, 119, 119, 0.699);
+	box-shadow: .1em -.1em 0 rgba(51, 51, 51, 0.681),
+				.2em -.2em 0 rgba(51, 51, 51, 0.712),
+				-.1em .1em 0 rgba(51, 51, 51, 0.716),
+				-.2em .2em 0 rgba(51, 51, 51, 0.655),
+				-.3em .3em 0 rgba(51, 51, 51, 0.67),
+				inset -.1em -.1em 2em #FFF,
+				-.1em -.1em 2em #777;
+}
+
+.button-inside {
+	width: 11em;
```

Comment:
this and other class  name also be like kkartik07-button-2-inside kkartik07-button-2-text

as if some one give that class in they buttons then other thing overwrite ( all )